airbyte-integrations/connectors/destination-snowflake/src/main/resources/spec.json

```diff
@@ -68,11 +68,17 @@
         "title": "Password",
         "order": 6
       },
+      "jdbc_url_params": {
+        "description": "Additional properties to pass to the jdbc url string when connecting to the database formatted as 'key=value' pairs separated by the symbol '&'. (example: key1=value1&key2=value2&key3=value3).",
+        "title": "JDBC URL Params",
+        "type": "string",
+        "order": 7
```

Comment:
Do you think you could add a `pattern` to validate with a regex that the structure of input value matches `key1=value1&key2=value2&key3=value3`

Reply:
Oh yea! I didn't know that spec existed.

Reply:
Actually, I'm not sure if requiring a regex here makes sense. We dont really have control over what a customer will need to put in as an additional JDBC parameter. 

For example:

If someone wants to specify a [password](https://docs.snowflake.com/en/user-guide/jdbc-configure.html#private-key-file-name-and-password-as-connection-properties) parameter. That string could have anything in it, and I dont think Airbyte should reject it if we had an alphanumeric regex requirement or something.

Reply:
I would also say that, if a customer does input something wrong, the connection test will spit back a failed connection error about an invalid string so the customers would know to fix it. I mostly took this from the MySQL Source which implements this feature and it does not validate a pattern

Reply:
Right, makes sense.